renderer/src/features/mcp-servers/components/dialog-form-run-mcp-command.tsx

```diff
@@ -102,6 +102,7 @@ export function DialogFormRunMcpServerWithCommand({
     <Dialog open={isOpen} onOpenChange={onOpenChange}>
       <DialogContent
         className="p-0 sm:max-w-2xl"
+        onCloseAutoFocus={() => form.reset()}
```

Comment:
I thought that not resetting the form on closing when no submission was made was something we wanted. We have the same behavior on the registry too

Reply:
yeah I don't know the reason, but in any case let's be consistent if we want to change it